keps/sig-storage/695-skip-permission-change/README.md

```diff
@@ -105,6 +106,37 @@ type PodSecurityContext struct {
 }
 ```
 
+#### Handling of volume types that apply gid as a mount option
+
+Handling of volume types that use fsGroup as a mount option is out of scope for 1.20 beta, but following section still describes the general mechanism that will be used.
```

Comment:
If we add the Mount policy later, then what should the older kubelet behavior be if it sees this new value?

Reply:
Now that I think of -  this might be better explained in @huffmanca KEP update and we need to ensure that we are doing this correctly now vs doing this in a future release. I will sync with @huffmanca offline.

Nonetheless I have made a note of that in this KEP.